lib/msal-browser/src/utils/BrowserUtils.ts

```diff
@@ -102,12 +100,18 @@ export function parseAuthResponseFromUrl(): {
     }
 
     if (!payload || !params) {
-        throw new Error("No auth payload found on URL (hash or query)");
+        throw new AuthError(
```

Comment:
Since this is using BrowserAuthErrorCodes this should be BrowserAuthError type
```suggestion
        throw createBrowserAuthError(
```